layer/mounted_layer.go

```diff
@@ -155,6 +156,12 @@ func (rl *referencedRWLayer) Mount(mountLabel string) (string, error) {
 
 	if rl.activityCount > 0 {
 		rl.activityCount++
+		path, err := rl.mountedLayer.layerStore.GetMountPathFromDisk(rl.mountedLayer.name)
```

Comment:
Updated diffs.

regarding your comment on "add it as a property of mountedLayer. Then it is cached for remounts and loaded on startup like the rest of the properties there.":

path is already a field in mountedLayer. Even if we add a redundant pathOnDisk field to mountedLayer, NewDaemon -> loadMount and CreateRWLayer-> saveMount (i.e mounts load and save) are called before the container.RWLayer.Mount(). So path cannot be treated like the other fields like mountID and initID for caching.